CHANGELOG.md

```diff
@@ -10,6 +10,17 @@ You can now style buttons on dark backgrounds to have a white background colour
 
 This change was made in [pull request #3556: Add inverse button styles](https://github.com/alphagov/govuk-frontend/pull/3556).
 
+#### New link styles are now enabled by default
+
+In GOV.UK Frontend v3.12.0 we introduced new link styles which:
+
+- have underlines that are consistently thinner and a bit further away from the link text
+- have a clearer hover state, where the underline gets thicker to make the link stand out to users
+
+The new link styles are now enabled by default. If you are setting `$govuk-new-link-styles` to `true` in your Sass you can now remove this.
+
+This change was made in [pull request #3599: Enable new link styles by default](https://github.com/alphagov/govuk-frontend/pull/3599).
+
```

Comment:
**question** Did we say this setting would be removed in the next breaking change? If that's the case, do we want to add a little note that the setting is deprecated?

Reply:
Would appreciate your thoughts on this in #3600! If we don't want to do #3600 then yes, we should probably deprecate it instead.

Reply:
Actually, I'll merge this and if we do decide to deprecate it we can do that in a follow up PR too.